plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1643,4 +1644,9 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
     initializeGesturesManager(gesturesManager, true)
     initializeGestureListeners(context, true)
   }
+
+  companion object {
```

Comment:
mark this companion object as private and rebase your PR on latest `main` to make CI pass.

Reply:
all the variables inside this is private,  so i hope that shouldn't affect anything. yeah will rebase :)

Reply:
probably it can be visible as empty comp object